src/main/webapp/encounters/manualAnnotation.jsp

```diff
@@ -300,6 +300,15 @@ try{
 		}
 		
 	}
+	else if(annots.size()==1 && !annots.get(0).isTrivial() && iaClass!=null &&  iaClass.indexOf("+")>-1){
+		//exception case - if there is only one annotation and it is a part
+		if(annots.size()==1){
```

Comment:
i think this conditional could be dropped, since it is already tested as part of the `&&` test in line 303.